modules/reference/pages/rpk/rpk-topic/rpk-topic-produce.adoc

```diff
@@ -39,6 +39,24 @@ Produce to `foo`, using schema ID 1, message FQN Person.Name:
 rpk topic produce foo --schema-id 1 --schema-type Person.Name
 ----
 
+=== Tombstones
+
+By default, records produced without a value will have an empty-string value, `""`. The below example produces a record with the key `not_a_tombstone_record` and the value `""`:
+
+```bash
+rpk topic produce foo -k not_a_tombstone_record
+[Enter]
+```
+    
+Tombstone records (records with a `null` value) can be produced by using the `-Z` flag and creating empty-string value records. Using the same example from above, but adding the `-Z` flag will produce a record with the key `tombstone_record` and the value `null`:
+
+```bash
+rpk topic produce foo -k tombstone_record -Z 
+[Enter]
+```
+
+It is important to note that records produced with values of string `null` are not considered tombstones by Redpanda.
```

Comment:
This should read,

>It is important to note that records produced with values of string \`"null"` are not considered tombstones by Redpanda.


Note the quotations around null- very important distinction.